lua/telescope/make_entry.lua

```diff
@@ -613,31 +613,39 @@ function make_entry.gen_from_packages(opts)
   end
 end
 
-function make_entry.gen_from_apropos()
+function make_entry.gen_from_apropos(opts)
+  local sections = {}
+  for _, section in ipairs(opts.sections) do
+    sections[section] = true
+  end
+
   local displayer = entry_display.create {
-    separator = "",
+    separator = ' ',
     items = {
       { width = 30 },
-      { remaining = true },
+      {},
```

Comment:
we should keep the `remaining = true` here after that i think we are done right?